### PR TITLE
FortiSoar for Slack SP

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
#### Descriptions:
Solution Pack for Slack Bot 

On installing the solution pack and slack application on slack, */fortisoar + command* can be used to invoke all the bot actions! 
eg. /fortisoar createAlert.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Release Information
+
+- **Version**: 2.0.0
```
Comment (cs-suyog-jinturkar) [inline README.md]: 2.0.0?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- **Certified**: Yes
+
+- **Publisher**: Fortinet  
+
+- **Compatibility**: 7.2.2 and later
```
Comment (cs-suyog-jinturkar) [inline README.md]: 7.2.2 or 7.3.1?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- **Applicable**: Dashboards
+
+- [Release Notes](./widget/release_notes.md)
```
Comment (cs-suyog-jinturkar) [inline README.md]: Is this path correct?

File: connectors/data.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    {
+        "name": "slack",
+        "label": "Slack",
+        "category": "Communication",
+        "description": "Slack is a cloud-based set of proprietary team collaboration tools and services.This connector facilitates automated operations like list channels, list users, send message etc.",
+        "publisher": null,
+        "operation_roles": [],
+        "configurations": [],
+        "dataImports": [],
+        "install_mode": "rpm"
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:11]: rpm, confirm?

File: connectors/data.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "name": "slack",
+        "label": "Slack",
+        "category": "Communication",
+        "description": "Slack is a cloud-based set of proprietary team collaboration tools and services.This connector facilitates automated operations like list channels, list users, send message etc.",
+        "publisher": null,
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:7]: publisher?

File: info.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
+{
+    "name": "fortiSOARForSlack",
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.2.0",
```
Comment (cs-suyog-jinturkar) [inline info.json]: 7.2.0?

File: info.json [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    "type": "solutionpack",
+    "local": true,
+    "label": "FortiSOAR For Slack",
+    "dependencies": [
+        {
+            "name": "sOARFramework",
+            "type": "solutionpack",
+            "label": "SOAR Framework",
+            "minVersion": null
```
Comment (cs-suyog-jinturkar) [inline info.json:13]: minVersion empty?

File: info.json [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Community",
+    "description": "FortiSOAR for Slack application allows teams to leverage the power of FortiSOAR as part of their daily communications.",
+    "help": null,
+    "category": [
+        "Communication"
+    ],
+    "supportInfo": null,
```
Comment (cs-suyog-jinturkar) [inline info.json]: supportInfo emptY?

File: connectors/data.json (hunk 5 above)
Comment (cs-yash-patodkar) [reply connectors/data.json:7]: Checked with rest of the solution packs, the publisher in connector's data.json is null, is generated by default on selecting a connector while creating a solution pack

File: connectors/data.json (hunk 4 above)
Comment (cs-yash-patodkar) [reply connectors/data.json:11]: Checked with rest of the solution packs, the install_mode is rpm, is generated by default on selecting a connector while creating a solution pack

File: info.json [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+            "name": "sOARFramework",
+            "type": "solutionpack",
+            "label": "SOAR Framework",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Community",
+    "description": "FortiSOAR for Slack application allows teams to leverage the power of FortiSOAR as part of their daily communications.",
+    "help": null,
```
Comment (cs-rohit-khune) [inline info.json:20]: please add doc link, also add "certified": "false", if this is uncertified release

File: info.json [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+    "dependencies": [
+        {
+            "name": "sOARFramework",
+            "type": "solutionpack",
+            "label": "SOAR Framework",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Community",
```
Comment (cs-rohit-khune) [inline info.json]: publisher should be "Fortinet"

File: picklists/Bot Channel.json [hunk 11]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (cs-akash-jarad) [inline picklists/Bot Channel.json]: change the file name to External channel and picklist name also

**Review by cs-suyog-jinturkar — APPROVED** [review]
WIP

**Pull request merged**